### ISSUE TITLE
will it bother anyone else that "generate" is missing an "e"?

### ISSUE THREAD

**Issue by podfeet** [bartificer/xkpasswd-js#68]
In passwordview.mjs:

```
constructor() {
    this.#passwordList = $('ul#generatpassword_presentation_password');
```

I'd fix it but for the life of me I can't figure out where this is used in which file(s).

**Comment by hepabolu** [comment]
It should match the <ul> element in the index.html

**Comment by podfeet** [comment]
<img width="978" alt="image" src="https://github.com/bartificer/xkpasswd-js/assets/12651464/43e677f1-e00a-4c3f-8285-93004a12db53">

**Comment by hepabolu** [comment]
So it’s a typo in the constructor and it should be fixed

**Comment by podfeet** [comment]
Agreed. but I don't know where all the places this is used. If it only exists where I pointed to it, then is it ever used at all?

**Comment by hepabolu** [comment]
This is odd. The constructor of the passwordview.mjs has the correct ul-id, which is different than the one you wrote down in the original post.
So the code available in GitHub is correct and you probably have a botched or old version.

**Comment by pricemi115** [comment]
Whew!! That would be "classic me" though 🤣
